yarn.lock

```diff
@@ -78,17 +78,16 @@
     source-map-support "0.5.21"
 
 "@appium/support@^2.55.3", "@appium/support@^2.57.0", "@appium/support@^2.59.3", "@appium/support@^2.60.0":
-  version "2.60.0"
-  resolved "https://registry.yarnpkg.com/@appium/support/-/support-2.60.0.tgz#3ab7cbc94a219d61aabe269d1336b078dabdeffb"
-  integrity sha512-hrCeBqeQ0xxAEGLxF/8CfwXTQMgD9z5Jd5Bz5JCFbmgSSRUd8u0MaeiRQJDfCB9q6m3Gw2KXFliFhe+M9Hf/WA==
+  version "2.61.0"
+  resolved "https://registry.yarnpkg.com/@appium/support/-/support-2.61.0.tgz#11b4335a3ce3c4d264d04183acecbf7c4ff1c435"
+  integrity sha512-xHsmB3hpw7IushPjfPFgI3xW1kD+f211hDCWQkKoxcv64oK3F5rqvKPzFfJQLYZi8uFewdfC2cyL7ApVdOeN0Q==
   dependencies:
-    "@appium/types" "^0.4.1"
-    "@babel/runtime" "7.19.0"
```

Comment:
Just wanted to check that removing this was intentional?

Reply:
Yeah that's based on what the package declares its dependencies are

This entire file is tool-generated. You can force things to be regenerated by deleting certain packages and rerunning yarn, which will prompt yarn to fill in the lock file entry.